src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -468,6 +469,8 @@ private TaskAgent UpdateExistingAgent(TaskAgent agent, RSAParameters publicKey,
             // update should replace the existing labels
             agent.Version = BuildConstants.RunnerPackage.Version;
             agent.OSDescription = RuntimeInformation.OSDescription;
+            agent.Ephemeral = ephemeral;
```

Comment:
Curious when the service will fully support the behavior - i.e. has the service code rolled out yet?

Also curious whether it will work on GHES? Did the server-side code make it into GHES. If not, curious if we can detect/error during runner config.

Also are we ready to document and forever support the new behavior?

Reply:
I confirmed the server changes did not make into 2.22

Reply:
note the service behavior will go into GA @ericsciple